README.md

```diff
@@ -2,63 +2,67 @@
 
 [![Build Status](https://travis-ci.org/deis/workflow.svg?branch=master)](https://travis-ci.org/deis/workflow) [![Go Report Card](http://goreportcard.com/badge/deis/workflow)](http://goreportcard.com/report/deis/workflow)
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide a lightweight, [Heroku-inspired](http://heroku.com) workflow.
+Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage
+applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
+a lightweight, [Heroku-inspired](http://heroku.com) workflow.
 
 ## Work in Progress
 
 ![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
 
-Deis Workflow v2 is changing quickly. Your feedback and participation are more than welcome, but be aware that this project is considered a work in progress.
+Deis Workflow v2 is changing quickly. Your feedback and participation are more than welcome, but be
+aware that this project is considered a work in progress.
 
 ## Hacking Workflow
 
-First, install [deis/etcd](https://github.com/deis/etcd) as described in its documentation. Ensure that the deis-etcd-service is running and healthy by accessing its port 4001 and seeing that the environment variable `DEIS_ETCD_1_SERVICE_HOST` is set.
+First, install [helm](http://helm.sh) to get started. Next, add the deis repository to your chart
+list:
 
-One way to test this by running commands from another pod in the same namespace:
+```console
+$ helm repo add deis https://github.com/deis/charts
+```
+
+Then, install Deis!
 
 ```console
-$ kubectl exec alpine -- env | grep DEIS_ETCD_1_SERVICE | sort
-DEIS_ETCD_1_SERVICE_HOST=10.247.187.217
-DEIS_ETCD_1_SERVICE_PORT=2380
-DEIS_ETCD_1_SERVICE_PORT_CLIENT=4100
-DEIS_ETCD_1_SERVICE_PORT_PEER=2380
-$ kubectl exec alpine -- curl -sS 10.247.187.217:4100/version
-{"etcdserver":"2.2.1","etcdcluster":"2.2.0"}
+$ helm install deis/deis
 ```
 
-Next build the deis/workflow image and push it to a Docker registry. The `$DEIS_REGISTRY` environment variable must point to a registry accessible to your Kubernetes cluster. You may need to configure the Docker engines on your Kubernetes nodes to allow `--insecure-registry 192.168.0.0/16` (or the appropriate address range).
+You can then monitor their status by running
 
 ```console
-$ make docker-build docker-push
+$ kubectl get pods --namespace=deis
 ```
 
-Finally create a PostgreSQL database and the Deis workflow service:
+Once this is done, you can SSH into the minion running the controller and run the following:
+
+```
+$ vagrant ssh minion-1
```

Comment:
We could link to https://github.com/deis/workflow/blob/master/docs/src/installing-deis/quickstart.md#choose-a-provider, which is basically a proxy to kubernetes' documentation.